### PR TITLE
Updated development section

### PR DESCRIPTION
**Pull request by andy-maier**
Ready for review, even though it still contains a few TBDs about the docs build on GutHub pages. We can finalize those once the docs build is established.

### CODE REVIEW

File: docs_source/development.rst [hunk 1]
```diff
@@ -68,60 +68,56 @@ list of valid Make targets and a short description of what each target does.
 Building the documentation
 --------------------------
 
-The ReadTheDocs (RTD) site is used to publish the documentation for the
-zhmc-ansible-modules project at http://zhmc-ansible-modules.readthedocs.io/
+The documentation for the ibm.zhmc Ansible Galaxy collection is published
+on GitHub pages at https://zhmcclient.github.io/zhmc-ansible-modules/.
 
-This page automatically gets updated whenever the ``master`` branch of the
+That page automatically gets updated whenever the ``master`` branch of the
```
Comment (sabyadi) [inline docs_source/development.rst:74]: Should we always go with master for the docs? Maybe we could point the GitHub Pages to the latest released version while the next one is being developed?

File: docs_source/development.rst [hunk 2]
```diff
@@ -68,60 +68,56 @@ list of valid Make targets and a short description of what each target does.
...
+
 In order to build the documentation locally from the Git work directory, issue:
 
 .. code-block:: text
 
     $ make docs
 
 The top-level document to open with a web browser will be
-``build/docs/html/index.html``.
+``docs/build/html/index.html``.
```
Comment (sabyadi) [inline docs_source/development.rst:86]: The top level file to open should be docs/index.html.

File: docs_source/development.rst [hunk 3]
```diff
@@ -68,60 +68,56 @@ list of valid Make targets and a short description of what each target does.
...
-    $ TESTCASES="TestInit or TestSet" make test  # py.test -k expressions are possible
+* ``make linkcheck`` - Check links in documentation
+* ``make test`` - Run unit and function tests with test coverage
+* ``make sanity`` - Run Ansible sanity tests (includes flake8, pylint, validate-modules)
+* ``make end2end`` - Run end2end tests (against a real environment)
 
-To run the unit tests and some more commands that verify the project is in good
-shape in all supported Python environments, use Tox:
+For the unit and function tests, the testcases and options for pytest
+can be specified via the nvironment variable ``TESTOPTS``, as shown in these
```
Comment (sabyadi) [inline docs_source/development.rst:104]: Typo in the work environment.

File: docs_source/development.rst [hunk 4]
```diff
@@ -172,42 +168,41 @@ has the remote name ``origin`` in your local clone.
...
+    * In the "Known issues" list item, remove the link to the issue tracker and
+      add text for any known issues you want users to know about.
+    * Remove all empty list items in the section of the version to be released.
 
 5.  Commit your changes and push them upstream:
 
     .. code-block:: text
 
-        git add docs/changes.rst
+        git add docs/source/changes.rst
```
Comment (sabyadi) [inline docs_source/development.rst:188]: Should be docs_source instead if docs/source.

File: docs_source/development.rst [hunk 5]
```diff
@@ -373,13 +363,13 @@ has the remote name ``origin`` in your local clone.
 
     .. code-block:: text
 
-        vi docs/changes.rst
+        vi docs/source/changes.rst
```
Comment (sabyadi) [inline docs_source/development.rst:366]: Same as above.

File: docs_source/development.rst [hunk 6]
```diff
@@ -266,23 +265,18 @@ has the remote name ``origin`` in your local clone.
...
-    Once that build is complete, verify again.
+    **TBD: Verify the above on GitHub pages once established**
 
 13. On GitHub, edit the new tag ``M.N.U``, and create a release description on it. This
     will cause it to appear in the Release tab.
 
     You can see the tags in GitHub via Code -> Releases -> Tags.
 
-14. Upload the package to PyPI:
+14. Publish the collection to Ansible Galaxy:
```
Comment (sabyadi) [inline docs_source/development.rst:279]: Shouldn't there be a step to modify the galaxy.xml to update it with the new version?

**Review by sabyadi — COMMENTED** [review]
Please have a look at my comments.

File: docs_source/development.rst (hunk 2 above)
Comment (andy-maier) [reply docs_source/development.rst:86]: Agreed. This was before that change and needs to be adjusted. Thanks for catching it.

File: docs_source/development.rst (hunk 1 above)
Comment (andy-maier) [reply docs_source/development.rst:74]: Ideally, we would have a docs on GitHub pages that supports multiple versions, like it is the case on RTD. Then we can point to the master, the latest released, and even the latest released fix version of the stable branch. I have created issue #211 for that. This text here has a TBD that asks for adjusting that once Github pages is in place, so we can merge this PR and solvbe that later. i have created issue #212 to not forget that.

File: docs_source/development.rst (hunk 3 above)
Comment (andy-maier) [reply docs_source/development.rst:104]: DONE

File: docs_source/development.rst (hunk 4 above)
Comment (andy-maier) [reply docs_source/development.rst:188]: DONE

File: docs_source/development.rst (hunk 5 above)
Comment (andy-maier) [reply docs_source/development.rst:366]: DONE

File: docs_source/development.rst (hunk 6 above)
Comment (andy-maier) [reply docs_source/development.rst:279]: Yes. :-)
DONE

Comment (andy-maier) [reply docs_source/development.rst:279]: Also in the releasing a version section, where itz now asks to verify the version in galaxy.yml.

**Comment by andy-maier** [comment]
Resolved all your comments, except the one about the master version, for which I created new issues.

**Pull request merged**